service/src/main/resources/application.yml

```diff
@@ -118,3 +118,24 @@ terra.common:
   tracing:
     stackdriverExportEnabled: ${env.tracing.exportEnabled}
     samplingRate: ${env.tracing.samplingRate}
+
+# Feature flags
+# The general rule is that feature flags should be set here in their production setting
+# and overridden in non-production environments.
+#
+# Feature flags can be overridden like any other Spring property. In particular, for local
+# execution, you can set an environment variable, such as:
+#
+#  FEATURE_AZURE_ENABLED=true
+#
+# The WSM helm chart will take all values under the "env" and make them environment variables
+# in the container, so you can get the equivalent setting by adding:
+#
+#  env:
+#    FEATURE_AZURE_ENABLED: true
+#
+# into the appropriate values file. That includes the values file in local_dev.
+feature:
```

Comment:
i'm imagining that it would become huge as all the feature flags will fall under here. Can we have sub feature defined here.

Reply:
We can have sub-features just by adding another layer to the YAML. For example,